<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>inteAnalyWidget</class>
 <widget class="QWidget" name="inteAnalyWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>665</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>246</y>
     <width>305</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"> #label_9 { 
	color: rgb(33, 131, 229);
    background-color: rgb(255, 255, 255);
  } 
 #label_9 * { 
     background-color:; 
 }</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeEdit_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>90</y>
     <width>194</width>
     <height>27</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="Chinese" country="China"/>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::PlusMinus</enum>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2016</year>
     <month>1</month>
     <day>1</day>
    </datetime>
   </property>
  </widget>
  <widget class="Line" name="line_6">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>164</y>
     <width>305</width>
     <height>2</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-top: 1px solid  rgb(33, 131, 229); 
background-color: rgb(33, 131, 229); </string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>620</y>
     <width>305</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"> #label_11 { 
    background-color: rgb(255, 255, 255);
  } 
 #label_11 * { 
     background-color:; 
 }</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_15">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>240</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-image: url(:/res/anniubk.png);
border-color: rgb(33, 131, 229);
border-width: 1px;
border-style: solid;</string>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_4">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>172</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>预警</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>告警</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>报警</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton_32">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>626</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(33, 131, 229);
border-color: rgb(33, 131, 229);
border-width: 1px;
border-style: solid;</string>
   </property>
   <property name="text">
    <string>报警</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>41</y>
     <width>305</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"> #label_3 { 
	color: rgb(0, 0, 0);
    background-color: rgb(255, 255, 255);
  } 
 #label_3 * { 
     background-color:; 
 }</string>
   </property>
   <property name="text">
    <string> 起始时间</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>170</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>4号车厢</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5号车厢</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>172</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>1车厢</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2车厢</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3车厢</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4车厢</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5车厢</string>
    </property>
   </item>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>667</y>
     <width>311</width>
     <height>2</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-top: 1px solid  rgb(33, 131, 229); 
background-color: rgb(33, 131, 229); </string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>625</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-image: url(:/res/anniubk.png);
border-color: rgb(33, 131, 229);
border-width: 1px;
border-style: solid;</string>
   </property>
   <property name="text">
    <string>故障图片下载</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>621</y>
     <width>702</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>164</y>
     <width>156</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"> #label_6 { 
	color: rgb(0, 0, 0);
    background-color: rgb(255, 255, 255);
  } 
 #label_6 * { 
     background-color:; 
 }</string>
   </property>
   <property name="text">
    <string> 车厢号</string>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>291</y>
     <width>305</width>
     <height>2</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-top: 1px solid  rgb(33, 131, 229); 
background-color: rgb(33, 131, 229); </string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_17">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>620</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/next_nor.png</normaloff>:/res/next_nor.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>41</width>
     <height>41</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="Line" name="line_7">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>40</y>
     <width>305</width>
     <height>2</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-top: 1px solid  rgb(33, 131, 229); 
background-color: rgb(33, 131, 229); </string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_5">
   <property name="geometry">
    <rect>
     <x>72</x>
     <y>207</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>火花放电</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>火花频率较高</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>火花较大</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>整体结构异常</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>7</y>
     <width>700</width>
     <height>606</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-image: url(:/res/playerbk.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>207</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-image: url(:/res/anniubk.png);
border-color: rgb(33, 131, 229);
border-width: 1px;
border-style: solid;</string>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_16">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>625</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-image: url(:/res/anniubk.png);
border-color: rgb(33, 131, 229);
border-width: 1px;
border-style: solid;</string>
   </property>
   <property name="text">
    <string>故障参数配置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_11">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>620</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/prev_nor.png</normaloff>:/res/prev_nor.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>41</width>
     <height>41</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>82</y>
     <width>305</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"> #label_4 { 
	color: rgb(0, 0, 0);
    background-color: rgb(255, 255, 255);
  } 
 #label_4 * { 
     background-color:; 
 }</string>
   </property>
   <property name="text">
    <string> 结束时间</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startTimeSetPushButton">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>52</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/res/time_set.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>133</y>
     <width>305</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"> #label_5 { 
	color: rgb(33, 131, 229);
    background-color: rgb(255, 255, 255);
  } 
 #label_5 * { 
     background-color:; 
 }</string>
   </property>
   <property name="text">
    <string> 位置及故障类型选择</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>197</y>
     <width>155</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"> #label_14 { 
	color: rgb(0, 0, 0);
    background-color: rgb(255, 255, 255);
  } 
 #label_14 * { 
     background-color:; 
 }</string>
   </property>
   <property name="text">
    <string> 故障类型</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="recordFileTableWidget">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>293</y>
     <width>305</width>
     <height>301</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: none;
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="showGrid">
    <bool>false</bool>
   </property>
   <property name="columnCount">
    <number>3</number>
   </property>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>100</number>
   </attribute>
   <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>选中</string>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="background">
     <color>
      <red>220</red>
      <green>220</green>
      <blue>220</blue>
     </color>
    </property>
   </column>
   <column>
    <property name="text">
     <string>序号</string>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="background">
     <color>
      <red>220</red>
      <green>220</green>
      <blue>220</blue>
     </color>
    </property>
   </column>
   <column>
    <property name="text">
     <string>文件名</string>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="background">
     <color>
      <red>220</red>
      <green>220</green>
      <blue>220</blue>
     </color>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>205</y>
     <width>305</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"> #label_8 { 
    background-color: rgb(255, 255, 255);
  } 
 #label_8 * { 
     background-color:; 
 }</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>291</y>
     <width>305</width>
     <height>331</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"> #label_10 { 
    background-color: rgb(255, 255, 255);
  } 
 #label_10 * { 
     background-color:; 
 }</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>2</y>
     <width>305</width>
     <height>46</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"> #label_2 { 
	color: rgb(33, 131, 229);
    background-color: rgb(255, 255, 255);
  } 
 #label_2 * { 
     background-color:; 
 }</string>
   </property>
   <property name="text">
    <string> 检索设置</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>158</x>
     <y>167</y>
     <width>153</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"> #label_7 { 
	color: rgb(0, 0, 0);
    background-color: rgb(255, 255, 255);
  } 
 #label_7 * { 
     background-color:; 
 }</string>
   </property>
   <property name="text">
    <string> 故障等级</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_33">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>626</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(33, 131, 229);
border-color: rgb(33, 131, 229);
border-width: 1px;
border-style: solid;</string>
   </property>
   <property name="text">
    <string>视频关联</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="EnddateEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>90</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="keyboardTracking">
    <bool>false</bool>
   </property>
   <property name="displayFormat">
    <string>yyyy-MM-dd</string>
   </property>
   <property name="date">
    <date>
     <year>2000</year>
     <month>2</month>
     <day>2</day>
    </date>
   </property>
  </widget>
  <widget class="QTimeEdit" name="StarttimeEdit">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>50</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>h时</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="EndtimeEdit">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>90</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>h时</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="StartdateEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="keyboardTracking">
    <bool>false</bool>
   </property>
   <property name="displayFormat">
    <string>yyyy-MM-dd</string>
   </property>
   <property name="date">
    <date>
     <year>2000</year>
     <month>2</month>
     <day>2</day>
    </date>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>215</x>
     <y>92</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>时间</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>215</x>
     <y>55</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>时间</string>
   </property>
  </widget>
  <widget class="QPushButton" name="alarmPushButton">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>630</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(33, 131, 229);
border-color: rgb(33, 131, 229);
border-width: 1px;
border-style: solid;</string>
   </property>
   <property name="text">
    <string>报警</string>
   </property>
  </widget>
  <widget class="QPushButton" name="canselPushButton">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>630</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(33, 131, 229);
border-color: rgb(33, 131, 229);
border-width: 1px;
border-style: solid;</string>
   </property>
   <property name="text">
    <string>注销</string>
   </property>
  </widget>
  <zorder>dateTimeEdit_2</zorder>
  <zorder>label_8</zorder>
  <zorder>label_9</zorder>
  <zorder>line_6</zorder>
  <zorder>label_11</zorder>
  <zorder>pushButton_15</zorder>
  <zorder>pushButton_32</zorder>
  <zorder>label_3</zorder>
  <zorder>comboBox</zorder>
  <zorder>line_4</zorder>
  <zorder>label_13</zorder>
  <zorder>label_6</zorder>
  <zorder>line_5</zorder>
  <zorder>pushButton_17</zorder>
  <zorder>line_7</zorder>
  <zorder>label_12</zorder>
  <zorder>pushButton_7</zorder>
  <zorder>pushButton_16</zorder>
  <zorder>pushButton_11</zorder>
  <zorder>label_4</zorder>
  <zorder>startTimeSetPushButton</zorder>
  <zorder>label_5</zorder>
  <zorder>label_14</zorder>
  <zorder>label_10</zorder>
  <zorder>label_2</zorder>
  <zorder>label_7</zorder>
  <zorder>pushButton_33</zorder>
  <zorder>comboBox_4</zorder>
  <zorder>comboBox_5</zorder>
  <zorder>comboBox_3</zorder>
  <zorder>pushButton_8</zorder>
  <zorder>recordFileTableWidget</zorder>
  <zorder>StarttimeEdit</zorder>
  <zorder>EndtimeEdit</zorder>
  <zorder>StartdateEdit</zorder>
  <zorder>EnddateEdit</zorder>
  <zorder>label_15</zorder>
  <zorder>label</zorder>
  <zorder>alarmPushButton</zorder>
  <zorder>canselPushButton</zorder>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
